public/js/app.js

```diff
@@ -0,0 +1,110 @@
+// Call function getNextMove(player, coord)
```

Comment:
All your top level functions and variables like 'computerMoves' and 'getNextMove' are global, so if a library you use has the same names, they could clash if the library is doing the same thing.  Always namespace by at least putting everything in a class like 

app = {
    computerMoves: []
}